server/user_account/tests.py

```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase
+from .models import CustomUser
+
+
+class CustomUserTestCase(TestCase):
+    def setUp(self):
+        CustomUser.objects.create(user_name="test_user", email="test@test.com", id=1, first_name='test',
+                                  last_name='user', role='SA')
+
+    def test_user_creation(self):
+        """ User was created correctly """
```

Comment:
Can you include this before the method the definition?

Reply:
The convention is to have it inside the method as I did.

Reply:
Should I proceed with the convention or write them on top

Reply:
Link me the convention?

Reply:
It's the PEP 257 Docstring Conventions -> https://www.python.org/dev/peps/pep-0257/

Reply:
[https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)
I guess I assumed Python had Java-like method commenting syntax, my bad.